code-ch11/tx.py

```diff
@@ -22,14 +22,14 @@ class TxFetcher:
     @classmethod
     def get_url(cls, testnet=False):
         if testnet:
-            return 'http://testnet.programmingbitcoin.com'
+            return 'https://blockstream.info/testnet/api/'
```

Comment:
you should move these into constants.

Reply:
Not a bad idea. I think to fix this I'd rather just make the minimum change here though.

Reply:
:+1: